td.vue/src/views/ImportModel.vue

```diff
@@ -126,7 +130,7 @@ export default {
                 } catch (e) {
                     // any error is most likely due to the picker being cancelled, which is benign so just warn
                     this.$toast.warning(this.$t('threatmodel.errors.open'));
-                    console.warn(e);
+                    // console.warn(e);
```

Comment:
we need the console error message here, because the toast warn says to refer to it